src/rosdiscover/config.py

```diff
@@ -8,6 +8,7 @@
 import attr
 import roswire
 import yaml
```

Comment:
bit of a nitpick that should be spotted by the linter: we split our import statements into three groups. First come the standard library imports, then third-party imports, and finally relative imports from our own package [https://www.python.org/dev/peps/pep-0008/#imports].

you should be able to run the linter locally while Travis misbehaves:
```
(someenv) $ tox
```

Reply:
tl;dr: add a new line between `import yaml` and `from .launch ...`

Reply:
done